quicklens/src/main/scala/com/softwaremill/quicklens/package.scala

```diff
@@ -47,20 +48,43 @@ package object quicklens {
     }
   }
 
-  implicit class QuicklensEach[F[_], T](t: F[T])(implicit f: QuicklensFunctor[F, T, T]) {
+  implicit class QuicklensEach[F[_], T](t: F[T])(implicit f: QuicklensEachFunctor[F, T]) {
```

Comment:
is this change necessary? Maybe not used now, but the functors can be kept general I suppose?